quic/s2n-quic-qns/src/client/perf.rs

```diff
@@ -40,7 +36,17 @@ pub struct Perf {
     local_ip: std::net::IpAddr,
 
     #[structopt(long, default_value)]
-    tls: TlsProviders,
```

Comment:
this was used to explicitly enable rustls vs s2n-tls. it breaks perf and it is also used in `run_endpoint.sh` which is used in interop to select the tls provider